actfw_core/application.py

```diff
@@ -61,20 +61,34 @@ def register_task(self, task):
             raise TypeError("type(task) must be a subclass of actfw_core.task.Task.")
         self.tasks.append(task)
 
-    def run(self):
+    def start(self):
         """Start application"""
         for task in self.tasks:
             task.start()
 
+    @property
+    def is_running(self):
+        return self.running
+
+    def run(self):
+        """Run & Wait application"""
+        self.start()
         try:
             while self.running:
                 time.sleep(1)
         except KeyboardInterrupt:
             pass
         except:
             raise
+        self.wait()
```

Comment:
I feel `self.wait()` should contain the above while loop.

Reply:
`self.wait` waits pipeline tasks in application. By this PR, I want to be able to execute other actions instead of a naive sleep loop between `start` and `wait`.

Reply:
https://github.com/Idein/actfw-core/pull/21/commits/49e933f40b6afc78251a6a968d6abd45a4e1e813